dot/sync/syncer.go

```diff
@@ -364,7 +364,12 @@ func (s *Service) handleBlock(block *types.Block) error {
 		}
 	} else {
 		logger.Debug("🔗 imported block", "number", block.Header.Number, "hash", block.Header.Hash())
-		telemetry.GetInstance().SendBlockImport(block.Header.Hash().String(), block.Header.Number)
+		telemetry.GetInstance().SendMessage(telemetry.NewTelemetryMessage(
+			telemetry.NewKeyValue("best", block.Header.Hash().String()),
+			telemetry.NewKeyValue("height", block.Header.Number.Uint64()),
+			telemetry.NewKeyValue("msg", "block.import"),
+			telemetry.NewKeyValue("origin", "NetworkInitialSync")))
+		// todo(ed) add timer to avoid a lot of sends
```

Comment:
does this cause a problem?

Reply:
also, is `block.import` used for all new builds (ie ones we built as well?) if so this might be better placed in `BlockState.AddBlock`

Reply:
Good point, it hasn't yet.